ST_main.c

```diff
@@ -591,6 +557,10 @@ static void init_args(int argc, char **argv) {
 				usage();
 			}
 
+			case 9: // -s (this option is obsolete. It is only present to allow users to get used to the change)
+			  printf("-s flag deprecated. STEPWAT2 now runs exclusively with SOILWAT2\n");
+				break;
+
```

Comment:
when might be a good time to remove this? I am thinking before we merge into master....